src/lib/components/EventScheduleModal/_styles.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../../styles/settings/colors';
+@import '../../../styles/tools/breakpoints';
+
+web-event-schedule-modal {
+  .modal {
+    border-radius: 4px;
+    box-shadow: 0 2px 4px rgba(0, 0, 0, .25);
+    padding: 16px 16px 8px;
+    background: white;
+    max-width: 100%; // modal puts us in 1em padding anyway
+    width: 512px;
+
+    @include bp(sm) {
+      padding: 16px 24px;
+    }
+  }
+
+  h2 {
+    margin: 0 0 12px;
+  }
+
+  // Modify the default button to match the mocks.
+  button.w-button.close {
+    letter-spacing: 0;
+    text-transform: none;
+    float: right;
+  }
+
+  p {
+    color: $GREY_600;
+    font: inherit;
+    font-size: 14px;
+    margin: 8px 0;
+
+    &:first-of-type {
+      margin-top: 16px;
+    }
+  }
+}
```

Comment:
i thought the sass linter grumped if files didn't end in a new line

Reply:
It does, but there's no extra line? Newlines are hard.